.github/workflows/ci.yml

```diff
@@ -0,0 +1,123 @@
+name: L0 smoke test regression suite
```

Comment:
Just a heads up - there's a [PR](https://github.com/chipsalliance/caliptra-rtl/pull/27/files#diff-81f00368f6e2ec134dea48176d97d18019089c9677b783b531d7e669e8828289) that's coming down the pipeline and based off the [caliptra-sw repo](https://github.com/chipsalliance/caliptra-sw/blob/main/.github/workflows/build-test-verilator.yml)

Since these are contained in two different files and can be pushed concurrently, I had a few comments below that are in the context of the links above.